.github/workflows/test.yml

```diff
@@ -0,0 +1,53 @@
+name: ci
+
+on:
+  - push
+  - pull_request
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+
+      - name: Install
+        run: pip install -r <(curl https://raw.githubusercontent.com/lsst/linting/master/requirements.txt)
+
+      - name: Run linter
+        run: flake8 --ignore=E133,E226,E228 --max-line-length=110 --max-doc-length=79 --exclude bin,doc,**/*/__init.py,**/*/version.py,tests/.tests
```

Comment:
Is this from the DM linting workflow template?

One thing that could be good long term is having a single source of truth for the flake8 configuration (and mypy too). You can put all this configuration into your `setup.cfg`, like: https://github.com/lsst/templates/blob/649807ecc7a7181f4bf6980946692b7ed9e09de1/project_templates/fastapi_safir_app/example/setup.cfg#L45-L58

That way this linting command will be consistent with `make lint`.

Reply:
Ah, yes, definitely better. Thanks!